SPECS/memcached/memcached.spec

```diff
@@ -52,6 +50,14 @@ Requires:       %{name} = %{version}-%{release}
 Install memcached-devel if you are developing C/C++ applications that require
 access to the memcached binary include files.
 
+%package        service
+Summary:        This package automatically runs scripts to set up memcached as a service.
+Requires(pre):  shadow-utils
+%{?systemd_requires}
```

Comment:
Doesn't this need to Require: memcached all up?  That way if you install memcached-service you get memcached the software component along with it?

Reply:
Good catch, adding that in. I had it that way because I was originally making `memcached` the service file and scripts and `memcached-exe` was going to be the binaries.